service/worker/scanner/tasklist/scavenger.go

```diff
@@ -154,6 +153,9 @@ func (s *Scavenger) run() {
 		s.stopWG.Done()
 	}()
 
+	// Start a task to delete orphaned tasks from the tasks table
+	s.executor.Submit(&orphanExecutorTask{scvg: s})
```

Comment:
Can you help to use a new config `worker.enableCleaningOrphanTaskInTasklistScavenger` and disable the behavior by default. 

Mostly because it could have some perf issue for some users and we don't want to introduce this by default.